Android/app/build.gradle

```diff
@@ -29,7 +29,7 @@ android {
             def prop = new Properties()
             prop.load(new FileInputStream(credsFile))
             buildConfigField('String', 'DITTO_APP_ID', prop['DITTO_APP_ID']) // always use single quotes here
-            buildConfigField('String', 'DITTO_OFFLINE_TOKEN', prop['DITTO_OFFLINE_TOKEN'])
+            buildConfigField('String', 'DITTO_PLAYGROUND_TOKEN', prop['DITTO_PLAYGROUND_TOKEN'])
```

Comment:
May need to pull in the Offline Token still since it is referenced in the code. Without it, we would see `Unresolved reference: DITTO_OFFLINE_TOKEN`. And vice versa for the Playground Token in release build.

Reply:
You are correct, `DITTO_OFFLINE_TOKEN` is used in the source code.

Reply:
I will be adding a commit soon to address it.